CDCgov/prime-reportstream
# 9778: add route message after all conversion is done

This PR moves sending the messages for the routing step to after all the FHIR bundles have been included

Test Steps:
This is a bug that occurred in the azure environment, so it's likely hard to manually verify.  I would recommend sending some test reports through the UP and there is a test that specifically covers the change

## Changes
- Sends the route messages after all the FHIR bundles have been converted

## Checklist

### Testing

### Process

## Linked Issues
- #9778 

## To Be Done

## Specific Security-related subjects a reviewer should pay specific attention to
